examples/observing_notebooks/AcquireAndTakeSequence.ipynb

```diff
@@ -0,0 +1,220 @@
+{
```

Comment:
<p>why is this part of the script? It's fine that it is, but maybe just add comments why you think this is useful (and the next cell).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/42/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>